sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/Customized/Models/MachineLearningPartialManagedServiceIdentity.Serialization.cs

```diff
@@ -84,15 +82,15 @@ internal static MachineLearningPartialManagedServiceIdentity DeserializeMachineL
             {
                 return null;
             }
-            ManagedServiceIdentityType managedServiceIdentityType = default;
+            Azure.ResourceManager.Models.ManagedServiceIdentityType managedServiceIdentityType = default;
```

Comment:
why not using the global using namespace instead of every usage?

Reply:
We kind of have to do this when this library generates a model with the same name.
If we do not do this, we will have an internal class `ManagedServiceIdentityType` generated as well, I do not like that.
We might need to fix that. When a type is replaced by a class in resourcemanager, we still write out that type that has been replaced, causing issues like this